setup.py

```diff
@@ -76,9 +77,14 @@ def compute_cmake_args():
             cmake_args += ['-DCMAKE_INSTALL_PREFIX=' + _ctx.cxx_install_root]
 
         else:
-            cxxLibDir = os.path.abspath(
-                os.path.join(setuptools.__file__, "../../opentimelineio/cxx-libs")
-            )
+            if "--user" in sys.argv:
+                cxxLibDir = os.path.abspath(
+                    os.path.join(_ctx.install_usersite, "opentimelineio/cxx-libs")
```

Comment:
To keep platform agnostic, this line should change to:
`os.path.join(_ctx.install_usersite, "opentimelineio", "cxx-libs")`